doc/distrib/NodeHelpFiles/DSRevitNodesUI.Categories.md

```diff
@@ -0,0 +1,6 @@
+## In Depth
+Categories contains a drop-down menu with every category available in a document.
```

Comment:
Please add ` ` around node names (and input/output port names) to help localization identify which strings should remain as is. For example, `Categories` contains...

Reply:
A blank space?